serialization_spec/serialization.py

```diff
@@ -56,8 +56,20 @@ def get_value(self, instance):
 
 
 class Filtered:
-    def __init__(self, filters, serialization_spec):
-        self.filters = filters
+    def __init__(self, *args):
+        if len(args) == 2:
+            self.field_name = None
+            self.filters, self.serialization_spec = args
+        elif len(args) == 3:
+            self.field_name, self.filters, self.serialization_spec = args
```

Comment:
This seems a bit odd. Could you do it with keyword arguments instead?

Reply:
You can't if you want the spec itself to be the last argument which I did want.

The implementation is a little odd but from the outside it looks nice. I think DRF has places where you can pass different arguments in different conditions?